### DISCUSSION TITLE
Purchase of ImageTrans Personel

### DISCUSSION THREAD

**Discussion by RageMist** [xulihang/ImageTrans-docs#491] [General]
I want to Purchase the ImageTrans Personel Version, I want to know if this Software is Active or Discontinued IF I face any, will it solved or not and is there any Coupon Codes I can use......I hope to get an Reply soon.

**Comment by xulihang** [comment]
Hi, you can learn about purchasing ImageTrans here: https://www.basiccat.org/imagetrans/#get-a-copy

**Reply by RageMist** [reply]
Thanks for the Reply now I know it it's an Active Software, but you didn't say anything about the Coupon part is there any coupon......and instead of using cards isn't there an option to use Paypal or something....

**Reply by xulihang** [reply]
Paypal is supported through fastspring. Coupon is only for college students.

**Reply by RageMist** [reply]
Well I do have my college ID if you want I can show it to you....and I don't see any Paypal option
![image](https://github.com/xulihang/ImageTrans-docs/assets/69992781/d0f512bb-1074-4ef4-b26a-d1e3375cd250)

**Comment by xulihang** [comment]
Please send it to my mail as listed on the webpage. Change the region to United States and you should be able to use Paypal.

**Reply by RageMist** [reply]
I sent the Email please check it...

**Reply by RageMist** [reply]
Is there something wrong with my Email.....I didn't get any reply.....

**Reply by RageMist** [reply]
Hello! Will it take time to get the coupon.....Even if I can't get the Coupon just tell me.....

**Reply by xulihang** [reply]
I've sent the coupon to you. Sorry about the delay. I had to go to sleep.
